lisa/tools/service.py

```diff
@@ -46,6 +46,9 @@ def check_service_status(self, name: str) -> bool:
     def check_service_exists(self, name: str) -> bool:
         return self._internal_tool._check_service_exists(name)  # type: ignore
 
+    def is_service_inactive(self, name: str) -> bool:
+        return self._internal_tool._is_service_inactive(name)  # type: ignore
```

Comment:
If there is no other implementation, just raise NotImplementError here, and overrides the method directly. Only if there are shared logic in base class, it needs a private method.

Reply:
the service class is not base class of ServiceInternal and Systemctl, it was created to hide/wrapper implement of ServiceInternal and Systemctl.